src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml

```diff
@@ -204,6 +204,62 @@
                 </span>
             </li>
         </ul>
+        @if (Model.Review.FilterType == ReviewType.PullRequest)
+        {
+            var associatedPRs = await Model.GetAssociatedPullRequest();
+            @if (associatedPRs != null && associatedPRs.Count() > 0)
+            {
+                var associatedPRState = String.Empty;
+                if (Request.Cookies.ContainsKey("associatedPRCollapse"))
+                {
+                    if (Request.Cookies["associatedPRCollapse"].Equals("shown"))
+                        associatedPRState = " show";
+                }
+                <p class="h6">
+                    <a data-bs-toggle="collapse" href="#associatedPRCollapse" aria-expanded="true" aria-controls="associatedPRCollapse">Associated Pull Requests&nbsp;&nbsp;<i class="fa-solid fa-ellipsis"></i></a>
+                </p>
+                <ul class="list-group collapse mb-3@(associatedPRState)" id="associatedPRCollapse">
+                    @foreach (var prModel in associatedPRs)
+                    {
+                        var url = $"https://github.com/{prModel.RepoName}/pull/{prModel.PullRequestNumber}";
+                        var txt = $"{prModel.RepoName}/{prModel.PullRequestNumber}";
+                        <li class="list-group-item">
+                            <a href="@url" target="_blank">@txt</a>
+                        </li>
                    }
+                </ul>
+            } 
+        }
+        @if (Model.Review.Language == "Swagger" && Model.Review.FilterType == ReviewType.PullRequest)

```

Comment:
Let's not limit this to swagger only.  We should support showing linked PR API reviews on any language. Similarly, it should show related API reviews from same PR for when they open even SDK level API reviews. 

For e.g. Assume that user opened .NET API review generated from swagger and he wants to see associated reviews like swagger, cadl, Python etc.